src/xmlstr2xmlnode.js

```diff
@@ -62,11 +63,39 @@ const props = [
   'tagValueProcessor',
   'attrValueProcessor',
   'parseTrueNumberOnly',
-  'stopNodes'
+  'stopNodes',
+  'preserveOrder'
 ];
 exports.props = props;
 
-const getTraversalObj = function(xmlData, options) {
+const nodeHasExtraText = (currentNode, tag) => {
+  return currentNode.parent && tag[14]
```

Comment:
We should pass value `tag[14]` to the function. So that method parameter name can be more meaningful.

Reply:
I gave it a name that is more explaining. Let me know what you think.